cassandra/row_parser.pyx

```diff
@@ -33,7 +34,16 @@ def make_recv_results_rows(ColumnParser colparser):
         desc = ParseDesc(colnames, coltypes, make_deserializers(coltypes),
                          protocol_version)
         reader = BytesIOReader(f.read())
-        parsed_rows = colparser.parse_rows(reader, desc)
+        try:
+            parsed_rows = colparser.parse_rows(reader, desc)
+        except Exception as e:
+            # Use explicitly the TupleRowParser to display better error messages for column decoding failures
+            rowparser = TupleRowParser()
+            reader.buf_ptr = reader.buf
+            reader.pos = 0
+            rowcount = read_int(reader)
+            for i in range(rowcount):
+                rowparser.unpack_row(reader, desc)
```

Comment:
The real fix is in the TupleRowParser class. The only reason this try/except block exists is in case some users are using custom parser ... written in Cython. Do they have any reason to do that? I'm not entirely sure it is worth to handle this case here and would be in favor to remove that part. What do you think?